app/src/main/java/org/pesmypetcare/mypetcare/activities/views/circularentry/forums/ForumsComponentView.java

```diff
@@ -0,0 +1,133 @@
+package org.pesmypetcare.mypetcare.activities.views.circularentry.forums;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.LinearLayout;
+
+import org.pesmypetcare.mypetcare.R;
+import org.pesmypetcare.mypetcare.activities.views.circularentry.CircularEntryView;
+import org.pesmypetcare.mypetcare.activities.views.circularentry.CircularImageView;
+import org.pesmypetcare.mypetcare.features.community.forums.Forum;
+import org.pesmypetcare.usermanager.datacontainers.DateTime;
+
+import java.util.List;
+
+public class ForumsComponentView extends CircularEntryView {
+    public static final int TEN = 10;
+    private Forum forum;
+
+    public ForumsComponentView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public ForumsComponentView(Context context, AttributeSet attrs, Forum forum) {
+        super(context, attrs);
+        this.forum = forum;
+    }
+
+    @Override
+    protected CircularImageView getImage() {
+        CircularImageView image = new CircularImageView(getCurrentActivity(), null);
+        Drawable groupDrawable = getResources().getDrawable(R.drawable.single_paw);
+
+        image.setDrawable(groupDrawable);
+        int imageDimensions = getImageDimensions();
+        image.setLayoutParams(new LinearLayout.LayoutParams(imageDimensions, imageDimensions));
+        int imageId = View.generateViewId();
+        image.setId(imageId);
+
+        return image;
+    }
+
+    @Override
+    public Object getObject() {
+        return forum;
+    }
+
+    @Override
+    protected String getFirstLineText() {
+        StringBuilder forumName = new StringBuilder(forum.getName());
+        String tags = getTags();
+
+        if (tags != null) {
+            forumName.append(' ').append(tags);
+        }
+
+        return forumName.toString();
+    }
+
+    @Override
+    protected String getSecondLineText() {
+        StringBuilder forumAuthorDate = new StringBuilder(getResources().getString(R.string.forum_created_on));
+        DateTime creationDate = forum.getCreationDate();
+        addText(forumAuthorDate, creationDate);
+
+        return forumAuthorDate.toString();
+    }
+
+    /**
+     * Add the text to display.
+     * @param forumAuthorDate The StringBuilder to add the text
+     * @param creationDate The creation date
+     */
+    private void addText(StringBuilder forumAuthorDate, DateTime creationDate) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method length is 17 lines (max allowed is 15).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5421176)